examples/python3/wt_session.py

```diff
@@ -0,0 +1,534 @@
+"""
+This module defines a class that manages an authenticated session
+with the WikiTree API.
+
+It allows you to log in and perform various queries.
+
+It requires python3 and the requests library.
+"""
```

Comment:
I would add here link for documentation, just to be complete https://github.com/wikitree/wikitree-api

Reply:
Have added URL as suggested.